sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Fixed proxy authentication type not being read from configuration. ([#28073](https://github.com/Azure/azure-sdk-for-java/issues/28073))
```

Comment:
This will have to be included in SB and EH. This change in AMQP itself doesn't fix the bug of not reading from config. This change only introduces the new constant for authentication type. Also, we should call out that SYSTEM_DEFAULTS will now use `NONE` as the default auth type.

Reply:
Have bugs to track the change in SB and EH, will add the note calling out the udpate to SYSTEM_DEFAULTS